pinot-common/src/main/java/org/apache/pinot/common/function/TransformFunctionType.java

```diff
@@ -87,7 +87,10 @@
 
   // Geo relationship
   ST_CONTAINS("ST_Contains"),
-  ST_EQUALS("ST_Equals");
+  ST_EQUALS("ST_Equals"),
+
+  // Geo indexing
+  GET_HEXAGON_ADDR("get_hexagon_addr");
```

Comment:
Suggest renaming it to `"geoToH3"`

Reply:
This func name has been used by presto at Uber, so try to be consistent

Reply:
The name should be `getHexagonAddr` for consistency